README.md

```diff
@@ -68,10 +68,10 @@ This GitHub repository is only used for issue tracking and making releases.
 Custom release builds
 ---------------------
 
-If a binary is not available for your platform, it's possibe to create a custom
+If a binary is not available for your platform, it's possible to create a custom
 build using the [Rust] toolchain. To do this, checkout the release tag for the
-version of interest and run `cargo build`. Alternatively the latest version may
-be built and installed from `crates.io` using `cargo install geckodriver`.
+version of interest and run `cargo build`. Alternatively, the latest version may
+be built and installed from `crates.io` using `cargo install geckodriver .
```

Comment:
This line should be reverted.

Reply:
Please note that the whole repository is a mirror and as such only used for making it easier for users to get the code in case they want to build geckodriver itself. The real content is at [hg.mozilla.org](https://hg.mozilla.org/mozilla-central/file/tip/testing/geckodriver/README.md).

But given that this is just a change for a comma I don't think it's worth to patching it immediately. I could take care of it when we do the next release of geckodriver.

Reply:
It's also fixing a typo in possible (guess same reasoning apply, just making sure that part was not missed).

Reply:
I'm not exactly sure which other typo you mean. The removal of the back tick is not correct given that it needs the ending character to have `cargo install geckodriver` highlighted as code.

Reply:
Change done in line 71.

Reply:
Oh, I overlooked that part. Thanks! I've now added a [comment to the 0.34.0 release tracking bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1824713#c1) for geckodriver so that I'll not forget about it.